pkgs/development/python-modules/readability-lxml/default.nix

```diff
@@ -31,8 +31,12 @@ buildPythonPackage rec {
     timeout-decorator
   ];
 
+  disabledTests = [
+    # Test is broken on darwin
```

Comment:
Could you please include the URL of the fixed PR in this comment?

Reply:
done